apm-agent-core/src/test/java/co/elastic/apm/agent/impl/ElasticApmTracerTest.java

```diff
@@ -338,7 +351,19 @@ void testPauseMidTransaction() {
     }
 
     @Test
-    void testSamplingNone() throws IOException {
```

Comment:
[for reviewer] this was the originally failing test that did not passed when executed in isolation. combined with the next modified block, it allows to cover the now removed test cases in other files.